contrib/kubernetes/manifests/registry-gcs-secret.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: gcs-keyfile
+type: Opaque
+data:
+  key.json: {base64 encode of the service account json file data}
+  bucket: {base64 encode of the buket name}
```

Comment:
bucket is same for any type of provide . So just asking if bucket and region can be set as a part of RC to provide better abstraction